setup.py

```diff
@@ -403,7 +403,7 @@ def run(self):
     url="https://github.com/huggingface/transformers",
     package_dir={"": "src"},
     packages=find_packages("src"),
-    package_data={"transformers": ["py.typed"]},
+    package_data={"transformers": ["py.typed", "*.cu", "*.cpp", "*.cuh"]},
```

Comment:
I think the `.h` files might be missing here as well?

Reply:
Pinging @Narsil here.

Reply:
You seem to use a `.h` file (src/transformers/models/deformable_detr/custom_kernel/cpu/ms_deform_attn_cpu.h) so yes you need `*.h` here too (if it's required at compile time which is most likely, otherwise you should just remove the file.